flang/docs/Directives.md

```diff
@@ -39,6 +39,8 @@ A list of non-standard directives supported by Flang
 * `!dir$ vector always` forces vectorization on the following loop regardless
   of cost model decisions. The loop must still be vectorizable.
   [This directive currently only works on plain do loops without labels].
+* `!dir$ ivdev` tells vecorisation to ignore the dependency analysis of the
```

Comment:
`ivdep`

`vectori[zs]ation`

Reply:
Darn it, missed the first one, fixed the spelling of optimization. Update coming in a few minutes.